README.md

```diff
@@ -1,8 +1,10 @@
 # Miden Rollup protocol
 
-<a href="https://github.com/0xPolygonMiden/miden-base/blob/main/LICENSE"><img src="https://img.shields.io/badge/license-MIT-blue.svg"></a>
-<img src="https://github.com/0xPolygonMiden/miden-base/workflows/CI/badge.svg?branch=main">
-<img src="https://img.shields.io/badge/rustc-1.75+-lightgray.svg">
+[![LICENSE](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/0xPolygonMiden/base/blob/main/LICENSE)
+[![test](https://github.com/0xPolygonMiden/base/actions/workflows/test.yml/badge.svg)](https://github.com/0xPolygonMiden/base/actions/workflows/test.yml)
+[![no-std](https://github.com/0xPolygonMiden/base/actions/workflows/no-std.yml/badge.svg)](https://github.com/0xPolygonMiden/base/actions/workflows/no-std.yml)
+[![RUST_VERSION](https://img.shields.io/badge/rustc-1.75+-lightgray.svg)]()
+[![CRATE](https://img.shields.io/crates/v/miden-base)](https://crates.io/crates/miden-base)
```

Comment:
There is no `miden-base` crate - so, this will show up as a broken link. Not sure what's the best way to handle this though. We could change this to show jus t`miden-objects`. Or show all 3 - `miden-objects`, `miden-lib`, and `miden-tx` - but this may be too busy. Or we could get rid of it completely. Or maybe there is another way?

Reply:
Fixed here: https://github.com/0xPolygonMiden/miden-base/pull/544